cmd/minishift/cmd/start.go

```diff
@@ -192,6 +192,10 @@ func runStart(cmd *cobra.Command, args []string) {
 		glog.Errorln("Error adding developer user to sudoers", err)
 		atexit.Exit(1)
 	}
+	if err := openshift.AddContextForProfile(constants.MachineName, ip); err != nil {
```

Comment:
This will set context as profile name instead hard coded `minishift` once we will have `MachineName` as env then user can get different context for different profiles.